README.md

```diff
@@ -90,6 +90,8 @@ When submitting a PR, please mark your PR with the "PR Ready for Review" label w
 
 To contribute to the `py2opsin` source code, start by cloning the repository (i.e. `git clone git@github.com:JacksonBurns/py2opsin.git`) and then inside the repository run `pip install -e .[dev]`. This will set you up with all the required dependencies to run `astartes` and conform to our formatting standards (`black` and `isort`), which you can configure to run automatically in vscode [like this](https://marcobelo.medium.com/setting-up-python-black-on-visual-studio-code-5318eba4cd00#:~:text=Go%20to%20settings%20in%20your,%E2%80%9D%20and%20select%20%E2%80%9Cblack%E2%80%9D.).
```

Comment:
Just noticed that the README references `astartes` rather than `py2opsin` here.

Reply:
Great catch, thanks! Just fixed it.